src/flash/com/longtailvideo/jwplayer/media/VideoMediaProvider.as

```diff
@@ -50,6 +50,10 @@ public class VideoMediaProvider extends MediaProvider {
     /** Is buffering due to load/seek or underflow? **/
     private var seeking:Boolean;
 
+    private var _preload:Boolean;
```

Comment:
1. Call `_stream.pause()` at the end of `init` after `loadQuality` (Then you don't have to call it at the end `loadStream`)
2. Call `setState(PlayerState.LOADING)` in `load` and `seek`, instead of `loadStream`
:boom: no more `_preload ` flag.

You can do the same with `_afterPreload`. Why do loadQuality and seek call loadStream? Why not just call play instead of calling loadStream if the item passes to init matches the item passed to load?